### PR TITLE
Style no_emails.html.

### PR DESCRIPTION
**Pull request by StevenXL**
- Remove incorrect / unused markup.
- Align text center only to h1.

### CODE REVIEW

**Comment by StevenXL** [comment]
Hello - I am unsure why the CircleCI build is failing. I suspect that it might be due to `MIX_ENV=prod mix compile --warnings-as-errors returned exit code 1`, as when I run the tests locally, I do get one warning. However, this is addressed on a separate pull request #184. I'm not sure how to proceed.

**Comment by paulcsmith** [comment]
@StevenXL I left a comment on #184. Thank you so much for tackling this :) Could you post a screenshot of what it looks like now?

File: lib/bamboo/plug/no_emails.html.eex [hunk 1]
```diff
@@ -24,43 +24,41 @@
...
       h1 {
         font-weight: 700;
         font-size: 18px;
       }
 
-      p {
-        color: #555;
-        line-height: 25px;
+      li {
+        margin-bottom: 20px;
```
Comment (paulcsmith) [inline lib/bamboo/plug/no_emails.html.eex]: I think we may want to lighten up the color to something like #666 or #777.

**Comment by paulcsmith** [comment]
The code looks good. After a screenshot I think we could finalize the last little bits :D Thanks again for your patience with the CI stuff. I think after a rebase it should pass CI

**Comment by StevenXL** [comment]
I am closing down this pull request in favor of [PR #188](https://github.com/thoughtbot/bamboo/pull/188), since it would take more time to fix the merge conflicts in this branch than it it worth.

**Pull request closed without merging**